WebApplication1/Models/Worker.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace WebApplication1.Models
+{
+    public class Worker
+    {
+        public int ID { get; set; }
+        [Required]
+        public string FirstName { get; set; }
+        [Required]
+        public string LastName { get; set; }
+        public Role Role { get; set; }
+
+    }
+
+    public enum Role
```

Comment:
It is always better to have enums and classes be in separate files, for better organization and accessibility. So ideally we should move this out in its own file and import it here